frontend/tests/topic.spec.ts

```diff
@@ -16,7 +20,7 @@ test.describe('Topic', () => {
         const fillText = 'example content ';
         const content = fillText.repeat((maxMessageSize / fillText.length) + 1);
 
-        const monacoEditor = page.locator('.monaco-editor').nth(1);
+        const monacoEditor = page.locator('.monaco-editor').nth(0);
```

Comment:
@rikimaru0345 this change was there because of the other change you're describing. We can as well provide more explicit test-id and search for a `monaco-editor` within that Locator. I'll investigate once I get to this.

Reply:
@rikimaru0345 fixed in https://github.com/redpanda-data/console/pull/1149/commits/0ef0a811c35d4ed29e05ad0f101cd69e414852a9